test/examples/universalBrowser.test.js

```diff
@@ -1,60 +1,62 @@
-/* globals expect */
-/* globals describe */
-/* globals it */
-/* globals sinon */
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import { setTimeout } from '../util/timers.js';
+import { loadHtml } from '../util/fixtures.js';
 
 describe('Rollbar loaded by snippet', function () {
-  before(function (done) {
-    // Stub the xhr interface.
-    window.server = sinon.createFakeServer();
+  let __originalOnError = null;
+
+  before(async function () {
+    // Prevent WTR/Mocha from failing the test on uncaught errors.
+    __originalOnError = window.onerror;
+    window.onerror = () => false;
 
     // Load the HTML page.
-    document.write(window.__html__['examples/universal-browser/test.html']);
+    await loadHtml('examples/universal-browser/test.html');
 
-    // Karma headless chrome won't dispatch DOMContentLoaded,
-    // so we need to do it manually.
-    var event = document.createEvent('Event');
-    event.initEvent('DOMContentLoaded', true, true);
-    document.dispatchEvent(event);
+    // DOMContentLoaded is not dispatched automatically in WTR/Playwright
+    document.dispatchEvent(new Event('DOMContentLoaded', { bubbles: true }));
 
     // Give the snippet time to load and init.
-    setTimeout(function () {
-      done();
-    }, 1000);
+    await setTimeout(250);
```

Comment:
1s wait is unnecessary and slows the test down, 10 is enough, left 250 just in case.